apps/services/user-profile/src/app/v2/dto/user-profileDto.ts

```diff
@@ -0,0 +1,45 @@
+import { IsBoolean, IsEnum, IsOptional, IsString } from 'class-validator'
+import { ApiPropertyOptional } from '@nestjs/swagger'
+
+import { DataStatus } from '../../user-profile/types/dataStatusTypes'
+
+export class UserProfileDto {
+  @ApiPropertyOptional()
```

Comment:
If property is not optional it should rather use `@ApiProperty()`.
With the new strategy of returning an empty profile we need to add some properties. We will discuss this offline.